__tests__/app.test.js

```diff
@@ -96,3 +97,55 @@ describe("/api/articles",()=>{
 })
 })
 
+describe("/api/articles/:article_id/comments",()=>{
+  test("GET 200: returns specifc article with all related comments",()=>{
+     return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body})=>{
+           expect(body.comments).toHaveLength(11);
+            body.comments.forEach((comment) => {
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
```

Comment:
As we're fetching all the comments related to article (article_id = 1), the assertion for article_id could be specific